test/model_simulation/simulate_SDEs.jl

```diff
@@ -119,31 +119,122 @@ let
     end
 end
 
+### Checks Simulations Don't Error ###
+
+# Tries to create a large number of problem, ensuring there are no errors (cannot solve as solution likely to go into negatives). 
+let
+    for reaction_network in reaction_networks_all
+        for factor in [1e-2, 1e-1, 1e0, 1e1]
+            u0 = factor * rand(rng, length(states(reaction_network)))
+            p = factor * rand(rng, length(parameters(reaction_network)))
+            prob = SDEProblem(reaction_network, u0, (0.0, 1.0), p)
+        end
+    end
+end
```

Comment:
What is this testing that isn't already tested? Don't we have lots of tests that create SDEProblems already?

Reply:
Also this isn't using the correct form of initial conditions and parameters, which need to be mappings going forward as we've discussed.